Automattic/course
# Rename page template to "Wide Width, No Title"

This PR renames the page template from "Page - No Title" to "Wide Width, No Title".

## Testing Instructions
- Create a page and assign it the "Wide Width, No Title" template.
- Add some different types of blocks to the page (e.g. paragraph, image etc.).
- View the page on the frontend.
- Ensure the page has no title and that it's wider than the default page template.
- Ensure there is adequate vertical space between blocks in the editor and on the frontend.

_Spacing Before_

![Screenshot 2022-11-10 at 11 41 20 AM](https://user-images.githubusercontent.com/1190420/201154808-ee61e5d3-f58a-4e38-8a2b-c3863a2390be.jpg)

_Spacing After_

![Screenshot 2022-11-10 at 11 41 50 AM](https://user-images.githubusercontent.com/1190420/201154931-c6bc99b6-c426-4219-a8cb-de111474057d.jpg)
